src/app/zap-templates/templates/app/cluster-objects.zapt

```diff
@@ -87,6 +88,22 @@ namespace {{asUpperCamelCase name}} {
 {{/last}}
 {{/zcl_structs}}
 
+{{#zcl_commands}}
```

Comment:
We shouldn't need to do this if the dependencies were correctly sorted in ZAP right? I thought Timotej had done this for structs at the very least...perhaps the linkage between request and response isn't as obvious.

If not achievable in this PR, would like to see if this can be fixed in ZAP so that we don't clutter this up here.

Reply:
If the commands were sorted in "responses come before their commands" order, yes, but it's not obvious that this is the "right" order in general....  And yes, doing that in this PR would be a bit of a pain, but we can look into it in a followup.